app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,9 @@
     <string name="app_name">Eater</string>
     <string name="recipes">Recipes</string>
     <string name="add_recipe">Add recipe</string>
+    <string name="LoginWelcome">Welcome to Eater!</string>
```

Comment:
sugestion: it is more common to use string keys like variables names in python, for example login_welcome